src/strawberry_sqlalchemy_mapper/mapper.py

```diff
@@ -259,7 +268,7 @@ def _convert_column_to_strawberry_type(
         corresponding strawberry type.
         """
         if isinstance(column.type, Enum):
-            type_annotation = column.type.python_type
+            type_annotation = strawberry.enum(column.type.python_type)
```

Comment:
From my reading of `strawberry.enum` it should be okay to wrap a type twice, but it would be great to have a test that does this. E.g. in my company, we already annotate all of our enums with @strawberry.enum, and I'm unsure if that would set `python_type` to the wrapped or unwrapped enum, and what would happen here.

Just a test that does
```
@strawbery.enum
class MyEnum(enum.StrEnum):
  FOO = "FOO"

class MyModel(Base):
  my_enum = Column(Enum(MyEnum))
```

Reply:
Another thought for the tests: what does that second wrapping do to any of the initial settings? e.g. If I `@strawberry.enum(name="DifferentName")` does it still respect the overwritten name?